nucleus/common/internal-api/src/main/java/org/glassfish/internal/deployment/analysis/TraceContext.java

```diff
@@ -96,6 +96,16 @@ public String getName() {
         return name;
     }
 
+    public String getLevelName(Level level) {
+        if (level == this.level) {
+            return name;
+        } else if (parent != null) {
```

Comment:
`else` cascade is not needed when using return. I know our formatting of always using curly brackets makes it longer if a sequence of individual `if` statements is used but I still think its more readable.